Bio/Nexus/Nexus.py

```diff
@@ -299,7 +299,7 @@ def _seqmatrix2strmatrix(matrix):
 
 
 def _compact4nexus(orig_list):
-    """Transform [1 2 3 5 6 7 8 12 15 18 20] (baseindex 0, used in the Nexus class)
+    r"""Transform [1 2 3 5 6 7 8 12 15 18 20] (baseindex 0, used in the Nexus class)
```

Comment:
Can you tell from the rest of the file if the next line should now say ``2-4 6-9 13-19\3 21`` rather than ``2-4 6-9 13-19\\3 21`` if you switch it to raw mode?